### ISSUE TITLE
needs-improvement : Sort local option

### ISSUE THREAD

**Issue by manu0600** [ranger/ranger#2522]
#### Runtime Environment
- Operating system and version: ArchLinux
- Terminal emulator and version: alacritty 0.9.0 (fed349aa)
- Python version: 3.9.9 (main, Nov 20 2021, 21:30:06)
- Ranger version/commit: ranger 1.9.3
- Locale: en_GB.UTF-8

#### Current Behavior
With the local option in `rc.conf` :
`setlocal path=~/Download sort mtime`,
the sorting cannot be changed anymore in this directory (by pressing `o` then any other key).

#### Expected Behavior
With this option in `rc.conf`, when navigating to the `~/Download`, then pressing `o` and `n` should sort by `natural`.

#### Context
Anytime I want to temporarily sort by name of size in this directory.

#### Possible Solutions
I didn't find a solution on my own yet.

#### Steps to reproduce
1. Add `setlocal path=~/Download sort mtime` in `rc.conf`
2. Restart Ranger
3. Navigate to `~/Download`
4. Press `o` and `n` : and it won't sort it by `natural`

#### Traceback
No crash

**Comment by toonn** [comment]
Yes, things like this are why we have (vague) plans to expand the filterstack functionality. We're currently thinking of providing a stack of default filters, like hidden files, a global stack that would apply on top of that in any directory and then a local stack on top of those that applies to the current directory.
